projects/chaos/test/test_power_path.c

```diff
@@ -62,34 +80,82 @@ static PowerPathCfg s_ppc = { .enable_pin = { .port = 0, .pin = 0 },
                                         .timer_id = SOFT_TIMER_INVALID_TIMER,
                                         .monitoring_active = false } };
 
+static CANStorage s_can_storage;
+static CANRxHandler s_rx_handlers[TEST_POWER_PATH_NUM_RX_HANDLERS];
+
 void setup_test(void) {
   event_queue_init();
-  gpio_init();
   interrupt_init();
-  gpio_it_init();
   soft_timer_init();
+  gpio_init();
+  gpio_it_init();
   adc_init(ADC_MODE_SINGLE);
+
+  CANSettings can_settings = {
+    .device_id = CAN_DEVICE_CHAOS,
+    .bitrate = CAN_HW_BITRATE_125KBPS,
+    .rx_event = CHAOS_EVENT_CAN_RX,
+    .tx_event = CHAOS_EVENT_CAN_TX,
+    .fault_event = CHAOS_EVENT_CAN_FAULT,
+    .tx = { GPIO_PORT_A, 12 },
+    .rx = { GPIO_PORT_A, 11 },
+    .loopback = true,
+  };
+
+  TEST_ASSERT_OK(
+      can_init(&can_settings, &s_can_storage, s_rx_handlers, SIZEOF_ARRAY(s_rx_handlers)));
   TEST_ASSERT_OK(power_path_init(&s_ppc));
 }
 
 void teardown_test(void) {}
 
 void test_power_path_uv_ov(void) {
+  volatile CANMessage rx_msg = { 0 };
+  can_register_rx_handler(CAN_MESSAGE_OVUV_DCDC_AUX, prv_handle_uvov, &rx_msg);
+
   TEST_ASSERT_OK(power_path_source_monitor_enable(&s_ppc.aux_bat, TEST_POWER_PATH_ADC_PERIOD_US));
   TEST_ASSERT_OK(power_path_source_monitor_enable(&s_ppc.dcdc, TEST_POWER_PATH_ADC_PERIOD_US));
 
+  delay_us(TEST_POWER_PATH_ADC_PERIOD_US + TEST_POWER_PATH_ADC_PERIOD_US / 10);
+
```

Comment:
Nice! You're using a GPIO interrupt to trigger the handler to run, allowing you to exercise that code.